### PR TITLE
feat(bin): Improve output

### LINKED ISSUE #9: Feature: better output
Current output looks a bit clumsy, can we make it better?

### PR DESCRIPTION
**Pull request by ta2edchimp**
Improve output of rules by rendering them in a tabular structure, like ls on *nixes.

Closes #9

### CODE REVIEW

**Comment by ta2edchimp** [comment]
Example Output:

image

File: src/bin/diff.js [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 'use strict'
 
+var size = require('window-size')
+var availableWidth = size.width || /* istanbul ignore next */ 80
+var ui = require('cliui')({width: availableWidth})
```
Comment (ta2edchimp) [inline src/bin/diff.js]: Available width for tabular display is calculated with respect to individual terminal (window) settings.

File: src/bin/find.js [hunk 2]
```diff
@@ -38,3 +56,9 @@ Object.keys(options).forEach(function findRules(option) {
 if (processExitCode) {
   process.exit(processExitCode)
 }
+
+function getOutputRuleCellMapper(width) {
```
Comment (ta2edchimp) [inline src/bin/find.js]: May be refactoring the function (duplicate, exists in `src/bin/diff.js` and `src/bin/find.js`) out of these two files, when tackling #7

**Comment by kentcdodds** [comment]
Oooohhh! I love it

**Comment by ta2edchimp** [comment]
Ha! At least when "diffing" it's not (yet) sorted alphabetically. Will fix ...

**Comment by ta2edchimp** [comment]
I think that's it for this PR ... the mentioned duplicate function should be refactored out when making changes to the overall output (according to #7).

Could you guys make a final review and maybe merge, when you got time for it?

File: src/lib/rule-finder.js [hunk 3]
```diff
@@ -71,21 +72,21 @@ function RuleFinder(specifiedFile) {
 
   // get all the current rules instead of referring the extended files or documentation
   this.getCurrentRules = function getCurrentRules() {
-    return currentRules
+    return sortRules(currentRules)
```
Comment (sarbbottam) [inline src/lib/rule-finder.js]: I would have named `sortRules` to `getSortedRules` as its is not sorting in-place but returning the sorted list.

File: src/bin/diff.js (hunk 1 above)
Comment (sarbbottam) [inline src/bin/diff.js]: awesome!! responsive ui 👍

File: src/bin/find.js (hunk 2 above)
Comment (sarbbottam) [inline src/bin/find.js]: If possible it will be great to see it along with this PR. Other wise just create an issue for cleaning up dups.

**Comment by sarbbottam** [comment]
Other than the `function name` I am sold out to this PR.

**Pull request merged**